test/framework/alertmanager.go

```diff
@@ -337,7 +345,7 @@ func (f *Framework) WaitForAlertmanagerConfigToContainString(ns, amName, expecte
 	})
 
 	if err != nil {
-		return fmt.Errorf("failed to wait for alertmanager config to contain %q: %v: %v", expectedString, err, pollError)
+		return fmt.Errorf("failed to wait for alertmanager config to contain %q: %v", expectedString, err)
```

Comment:
I don't think we need the pollError here as we already have `expectedString` inside the returned error.